### PR TITLE
MAISTRA-1611 Document injectPodRedirectAnnot field

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: topics/docs/comparison-with-istio/_index.adoc [hunk 1]
```diff
@@ -40,9 +40,33 @@ link:{{< relref "automatic-injection.adoc" >}}[the automatic injection section].
...
+
+==== Using Istio CNI with other Multus CNI plugins
+
+By default, if a pod contains an existing `k8s.v1.cni.cncf.io/networks` annotation,
+such as when using Multus CNI to add a macvlan network to the pod, the value of
+the annotation is overwritten. To preserve the value and instead append Istio CNI
+to the end, the field `spec.istio.sidecarInjectorWebhook.injectPodRedirectAnnot`
+must be set to `true` in the `ServiceMeshControlPlane` object as shown in the
+following example.
+
```
Comment (dgn) [inline topics/docs/comparison-with-istio/_index.adoc:59]: as it came up yesterday, should we mention that we only support the flat notation, not the [json notation](https://intel.github.io/multus-cni/doc/how-to-use.html#lauch-pod-with-json-annotation) of the annotation?

**Review by dgn — APPROVED** [review]
lgtm, just one comment

File: topics/docs/comparison-with-istio/_index.adoc (hunk 1 above)
Comment (luksa) [reply topics/docs/comparison-with-istio/_index.adoc:59]: Good point. I've now added a notice about this.

Comment (dgn) [reply topics/docs/comparison-with-istio/_index.adoc:59]: Thank you!

Comment (luksa) [reply topics/docs/comparison-with-istio/_index.adoc:59]: Since we now support the json notation, I've now updated this notice.

**Review by jwendell — APPROVED** [review]

File: vars.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // SMCP defaults
 //Value maistra spec.istio.global.tag
-:defaultMaistraTag: 1.1.2
+:defaultMaistraTag: 1.1.3
```
Comment (brian-avery) [inline vars.adoc]: Should have a Maistra release for this as well:

https://github.com/maistra/maistra.github.io/blob/maistra-1.2/data/releases/1.1.1.yaml

**Pull request merged**